api/src/main/java/org/openmrs/PatientProgramAttribute.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs;
+
+import org.openmrs.attribute.Attribute;
+import org.openmrs.attribute.BaseAttribute;
+
+public class PatientProgramAttribute extends BaseAttribute<ProgramAttributeType, PatientProgram> implements Attribute<ProgramAttributeType, PatientProgram> {
+    private Integer patientProgramAttributeId;
+
+    @Override
+    public Integer getId() {
+        return getPatientProgramAttributeId();
+    }
+
+    @Override
+    public void setId(Integer id) {
+        setPatientProgramAttributeId(id);
+    }
+
+    public PatientProgram getPatientProgram() {
+        return getOwner();
+    }
+
+    public void setPatientProgram(PatientProgram patientProgram) {
+        setOwner(new PatientProgram());
```

Comment:
Did you intentionally ignore the patientProgram parameter?

Reply:
That's directly from the bahmni source

Reply:
Does that mean that even the bahmni source ignored that parameter?